packages/backend/src/config/app.ts

```diff
@@ -137,3 +138,21 @@ function parseRedisTlsConfig(
 
   return Object.keys(options).length > 0 ? options : undefined
 }
+
+function parseOrProvisionKey(keyPath: string): crypto.KeyObject {
+  if (keyPath !== '') {
+    try {
+      const key = crypto.createPrivateKey(readFileSync(keyPath))
```

Comment:
Should this include support for reading an encrypted key file?

Reply:
Yes, probably. I'll include that

Reply:
Now that I thought more about it, I'm not so sure. We would also have to specify which encryption mechanism to use. And those key files are usually not encrypted. 🤔

Reply:
I'm fine leaving it out for now.

Reply:
#642